src/CMakeLists.txt

```diff
@@ -10,12 +9,11 @@ endif()
 link_directories(${MAMA_ROOT}/lib)
 link_directories(${MAMA_ROOT}/lib/dynamic)
 link_directories(${MAMA_ROOT}/lib/dynamic-debug)
-link_directories(${GTEST_ROOT}/lib)
 
 add_definitions(-DBRIDGE -DMAMA_DLL -DOPENMAMA_INTEGRATION)
 
 add_library(mamaomnmmsgimpl
-            MODULE Field.cpp

```

Comment:
Why shared and not module? From what I understand it should be MODULE if it's dlopened.

Reply:
You're right -- SHARED sets the 'x' bit on the library, but as it turns out that is not necessary.  (Although that was apparently a convention for some OS's at one point: https://unix.stackexchange.com/questions/400187/why-should-or-should-not-shared-libraries-be-executable-e-g-red-hat-vs-debian)

Will revert that.